AddOns/BasicFinancials/app/src/codeunit/ExperienceTier.Codeunit.al

```diff
@@ -0,0 +1,58 @@
+codeunit 20602 "Experience Tier BF"
+{
+    Access = Internal;
+
+    [EventSubscriber(ObjectType::Codeunit, Codeunit::"Application Area Mgmt. Facade", 'OnValidateApplicationAreas', '', true, true)]
+    local procedure OnValidateApplicationAreas(ExperienceTierSetup: Record "Experience Tier Setup"; TempApplicationAreaSetup: Record "Application Area Setup")
+    var
+    begin
+        if not ExperienceTierSetup."BF Basic Financials" then
+            exit;
+
+        TempApplicationAreaSetup.TestField("BF Orders", false);
+        TempApplicationAreaSetup.TestField("BF Basic Financials", true);
+    end;
+
+    [EventSubscriber(ObjectType::Codeunit, Codeunit::"Application Area Mgmt. Facade", 'OnSetExperienceTier', '', true, true)]
+    local procedure SetApplicationAreas(ExperienceTierSetup: Record "Experience Tier Setup"; var ApplicationAreasSet: Boolean; var TempApplicationAreaSetup: Record "Application Area Setup")
+    var
+        AppAreaMgmt: Codeunit "App Area Mgmt BF";
+    begin
+        if not ExperienceTierSetup."BF Basic Financials" then
+            exit;
+
+        AppAreaMgmt.GetEssentialExperienceAppAreas(TempApplicationAreaSetup);
+        DisableExperienceAppAreas(TempApplicationAreaSetup);
+        GetBasicFinancialsExperienceAppAreas(TempApplicationAreaSetup);
+        ApplicationAreasSet := true;
+    end;
+
+    local procedure GetBasicFinancialsExperienceAppAreas(var TempApplicationAreaSetup: Record "Application Area Setup" temporary)
+    var
+    begin
+        TempApplicationAreaSetup."BF Orders" := false;
+        TempApplicationAreaSetup."BF Basic Financials" := true;
+    end;
+
+    local procedure DisableExperienceAppAreas(var TempApplicationAreaSetup: Record "Application Area Setup" temporary)
+    var
+    begin
+        TempApplicationAreaSetup."Relationship Mgmt" := false;
```

Comment:
Warehouse and Jobs are still enabled?

Reply:
The previous analysis was done in collaboration with Peter Jacobsen (Microsoft) and various Microsoft resources.
Based on our analysis we have concluded, that Warehouse and Jobs is enabled in the Basic Financials license. Please reconcile it internally in Microsoft and then notify if you want to removed Warehouse and Jobs.